sklearn/linear_model/tests/test_logistic.py

```diff
@@ -245,20 +247,20 @@ def test_consistency_path():
     f = ignore_warnings
     # can't test with fit_intercept=True since LIBLINEAR
     # penalizes the intercept
-    for method in ('lbfgs', 'newton-cg', 'liblinear'):
+    for method in ('lbfgs', 'newton-cg', 'liblinear', 'sag'):
         coefs, Cs = f(logistic_regression_path)(
-            X, y, Cs=Cs, fit_intercept=False, tol=1e-16, solver=method)
```

Comment:
It is just to speed up the test, from `0.545s` to `0.155s`.
The tol can be lowered to 1e-3 if we accept to check only 3 decimal instead of 4. The test would last `0.102s`, but I don't want to decrease to much not to affect stability.

Without 'sag', the tests used to last `0.501s`